presto-docs/src/main/sphinx/release/release-0.147.rst

```diff
@@ -11,6 +11,7 @@ General Changes
 Hive Changes
 ------------
 
+* Fix reading symlink file format when the symlink crosses filesystem boundaries.
```

Comment:
Maybe phrase as "Fix reading symlinks when the target is in a different HDFS instance".